api/_lighthouse_ide_helper.php

```diff
@@ -1,7 +1,8 @@
 <?php
-# File generated by "php artisan lighthouse:ide-helper".
-# Do not edit this file directly.
-# This file should be ignored by git as it can be autogenerated.
+
+// File generated by "php artisan lighthouse:ide-helper".
```

Comment:
This file should have been excluded from the fixes, right?

Reply:
Oops, looks like since it does run form the root that is where the config also needs to be. Fixed in:

- [move pint config to root, fix lighthouse helper](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7753/commits/1dac3015074492d0b9c8af773c96d1a50cd8646a)
- [update workflow to run from root for config](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7753/commits/44a3b8e7a126673a5bee8a47dbca86bedf86ea18)

Reply:
Nice!  Maybe it's my imagination but it seems much faster from here, too.